godtools/ViewControllers/Tract/TractViewController.swift

```diff
@@ -84,7 +85,10 @@ class TractViewController: BaseViewController {
     
     override func configureNavigationButtons() {
         self.addHomeButton()
-        self.addShareButton()
+        
+        if self.resource?.code != "kgp-us" {
```

Comment:
Can you create a class property that contains a list of items we should ignore, and then just do a lookup on that array instead of checking one value? That will make it easier to add remove things as new tools get added to the app or the website.